_posts/en/newsletters/2022-12-21-newsletter.md

```diff
@@ -0,0 +1,798 @@
+[summarized][news181 rbf] a proposal by Jeremy Rubin to allow any
+transaction to be replaced by a higher fee alternative for a brief while
+after the original transaction was first seen by a node.  After that time had passed, the
+existing rules would apply---only allowing replacement of transactions
+opting in to [BIP125][].   This could allow merchants to accept
+unconfirmed transactions like they do now after the replacement time
+elapsed. More importantly, it may allow protocols that depend on
+replaceability for security to not have to worry about non-opt-in
+transactions as long as a protocol node or watchtower has a reasonable opportunity to
+respond soon after first learning of a transaction.
+
+At the end of January, Gloria Zhao started a fresh discussion about RBF
+by [posting][news186 rbf] background on the current RBF policy,
+enumerating several problems discovered with it over the years (such
+as [pinning attacks][topic transaction pinning]), an examination of how
+the policy affects wallet user interfaces, and the description of
+several possible improvements.  In early March, Zhao followed up with
+the [summary][news191 rbf] of two discussions about RBF between many
+developers, one discussion in person and the other online.
+
+Also in March, Larry Ruane raised [questions][news193 witrep] related to
+RBF about replacing transaction witnesses without changing the parts
+of a transaction from which its txid is derived.
+
+In June, Antoine Riard [opened][news205 rbf] a pull request to Bitcoin
+Core to add a `mempoolfullrbf` configuration option to Bitcoin Core.
+The option defaulted to Bitcoin Core's previous behavior of only
+allowing unconfirmed transactions to be replaced if they contained the
+[BIP125][] signal.  Nodes which were configured with the option set to
+its alternative value would accept transactions for relay and mining
+even if they replaced transactions that did not contain the BIP125
+opt-in signal.  Riard also started a thread on the Bitcoin-Dev mailing
+list to discuss the change.  Almost all pull request comments were
+positive and most mailing list discussion was about unrelated topics, so
+it was unsurprising that the pull request was [merged][news208 rbf]
+about a month after it was opened.
+
+In October, the Bitcoin Core project began distributing release
+candidates for version 24.0, which would be the first to include the
+`mempoolfullrbf` configuration option.  Dario Sneidermanis saw the draft
+release notes about the option and [posted][news222 rbf] to the
+Bitcoin-Dev mailing list that too many users and miners enabling the
+option would make unsignaled replacement reliable.  More reliable
+unsignaled replacement would also make it more reliable to steal from
+services that accept unconfirmed transactions as final, requiring those
+services to change their behavior.  Discussion [continued][news223 rbf]
+the next week and the [week after][news224 rbf].  A month after
```

Comment:
I think it could be worth mentioning that pull requests were opened in response to Dario's concerns sooner than 1 month later (https://github.com/bitcoin/bitcoin/pull/26287 and https://github.com/bitcoin/bitcoin/pull/26323). It just currently reads (to me) that no PRs were proposed until a month later.

```suggestion
the next week and the [week after][news224 rbf],
and several pull requests were opened to Bitcoin Core to restrict
or exclude the option in the 24.0 release. A month after
```